common/views/components/SearchTabs/SearchTabs.tsx

```diff
@@ -86,11 +86,17 @@ type Props = {
   activeTabIndex?: number;
   shouldShowFilters: boolean;
   showSortBy: boolean;
-  disableLink?: boolean;
   worksFilters: Filter[];
   imagesFilters: Filter[];
 };
 
+function submit(form: RefObject<HTMLFormElement>) {
+  form.current &&
+    form.current.dispatchEvent(
+      new window.Event('submit', { cancelable: true })
+    );
+}
+
```

Comment:
It seems a little like a leaky abstraction to have the search form controlled out here instead of in the `SearchForm`. I can't really figure out why we're doing it - could you explain?

Reply:
It is, but I needed a way to submit the form when the tabs change. N.B the SearchForm is only ever used inside SearchTabs

Reply:
Why do we submit the form when the tavbs change?

If that's the case, could we let the search for make it's submit, or formRef accessible?

Reply:
It's the current and expected behaviour. Yes, I went back and forth on making the formRef available, but it does make more sense to do that I think, so I'll do that